projects/aca-content/src/lib/components/favorite-libraries/favorite-libraries.component.spec.ts

```diff
@@ -99,6 +99,17 @@ describe('FavoriteLibrariesComponent', () => {
       expect(component.pagination).toBe(null);
       expect(component.isLoading).toBe(false);
     });
+
+    it('should set title based on selectedRowItemsCount', () => {
+      fixture.detectChanges();
+      const titleElement = fixture.nativeElement.querySelector('.aca-page-title');
```

Comment:
please see that comment, similar situation: 
https://github.com/Alfresco/alfresco-ng2-components/pull/9861#discussion_r1652573848

Reply:
added separated comment: https://github.com/Alfresco/alfresco-content-app/pull/3908/files#r1658375643